src/container/srv_cli.c

```diff
@@ -103,8 +104,7 @@ dsc_cont_open(daos_handle_t poh, uuid_t cont_uuid, uuid_t coh_uuid,
 		D_GOTO(out, rc = -DER_NOMEM);
 
 	/** destroyed in dsc_cont_close */
-	rc = dsc_cont_csummer_init(&cont->dc_csummer, pool->dp_pool,
-				   cont_uuid);
+	rc = dsc_cont_csummer_init(&cont->dc_csummer, pool->dp_pool, cont_uuid);
```

Comment:
Again, this is whitespace only isn't it?

Reply:
yes, whitespace only change.